packages/reactive-dict/reactive-dict.js

```diff
@@ -27,7 +27,10 @@ ReactiveDict = function (dictName) {
       this.name = dictName;
     } else if (typeof dictName === 'object') {
       // back-compat case: dictName is actually migrationData
-      this.keys = dictName;
+      this.keys = {};
+      for (var key in dictName) {
```

Comment:
We should probably check to make sure the iteration over `dictName` is only including the objects own properties. So adding in a `hasOwnProperty` check, or leveraging `Object.entries` like:

```
for (let [key, value] of Object.entries(dictName)) { 
  this.keys[key] = stringify(value);
}
```